packages/pharos/src/components/toggle-button-group/pharos-toggle-button.ts

```diff
@@ -0,0 +1,88 @@
+import { html, property } from 'lit-element';
+import type { TemplateResult, CSSResultArray, PropertyValues } from 'lit-element';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { toggleButtonStyles } from './pharos-toggle-button.css';
+import { designTokens } from '../../styles/variables.css';
+import { customElement } from '../../utils/decorators';
+import { PharosButton } from '../button/pharos-button';
+import type { ButtonType, LinkTarget, IconName } from '../button/pharos-button';
+
+export type { ButtonType, LinkTarget, IconName };
+
+/**
+ * Pharos toggle button component.
+ *
+ * @element pharos-toggle-button
+ *
+ * @slot - Contains the content of the button (the default slot).
+ *
+ */
+@customElement('pharos-toggle-button')
+export class PharosToggleButton extends PharosButton {
+  /**
+   * Indicates that the button is currently toggled on and cannot be pressed or focused by the user.
+   * @attr selected
+   */
+  @property({ type: Boolean, reflect: true })
+  public selected = false;
+
+  constructor() {
+    super();
+    this.variant = 'secondary';
+    this.type = 'button';
+  }
+
+  public static get styles(): CSSResultArray {
+    return [designTokens, toggleButtonStyles];
```

Comment:
```suggestion
    return [designTokens, super.styles, toggleButtonStyles];
```
You can also apply the Pharos button's styles here to avoid needing to duplicate them in the Sass file. Then you can simply override the styles you are interested in.

Reply:
done